core/domain/question_domain_test.py

```diff
@@ -564,71 +539,21 @@ def test_to_dict(self):
 
         self.assertEqual(expected_object_dict, self.observed_object.to_dict())
 
-    def test_validation_with_valid_properties(self):
+    def test_validation_with_valid_properties(self) -> None:
         self.observed_object.validate()
 
-    def test_validation_with_invalid_id(self):
-        self.observed_object.id = 1
-        with self.assertRaisesRegexp(
-            utils.ValidationError, 'Expected id to be a string, received 1'):
-            self.observed_object.validate()
-
-    def test_validation_with_invalid_interaction_id(self):
-        self.observed_object.interaction_id = 1
-        with self.assertRaisesRegexp(
-            utils.ValidationError,
-            'Expected interaction id to be a string, received 1'):
-            self.observed_object.validate()
-
-    def test_validation_with_invalid_question_content(self):
+    def test_validation_with_invalid_question_content(self) -> None:
         self.observed_object.question_content = 1
-        with self.assertRaisesRegexp(
+        # TODO(#xx2): Remove the type ignore[no-untyped-call] when file test_utils is fully type-annotated.
```

Comment:
For `no-untyped-call` you don't need to add a TODO.

Reply:
done